povm_toolbox/quantum_info/multi_qubit_frame.py

```diff
@@ -32,8 +35,10 @@
 
 from .base import BaseFrame
 
+LabelMultiQubitT = TypeVar("LabelMultiQubitT", int, tuple[int, ...])
```

Comment:
Please add a docstring to this object. Something like the following:
```suggestion
LabelMultiQubitT = TypeVar("LabelMultiQubitT", int, tuple[int, ...])
"""Some explanations of what this type variable implies...
"""
```